kernel/src/table_features/mod.rs

```diff
@@ -48,6 +51,9 @@ pub enum ReaderFeatures {
     /// vacuumProtocolCheck ReaderWriter feature ensures consistent application of reader and writer
     /// protocol checks during VACUUM operations
     VacuumProtocolCheck,
+    #[serde(untagged)]
+    #[strum(to_string = "{0}")]
+    Unknown(String),
```

Comment:
so this lets us parse unknown features - can you add docs for this and add tests?

Reply:
Haven't thoroughly tested this, but i think if we take

```rs
    #[strum(default)]
    Unknown(String),
```

here, we can avoid some of the custom implementation to parse into unkown types. At this point though i think order matters as well, and we need `Unknown` to be the last defined variant, which should then be a comment :).

Reply:
You can do that but it breaks the loseless-ness of this. giving you `Unknown("something")` instead of just `"something"`

Reply:
Probably missing something obvious, but why is that a problem and what are we loosing?

I guess we must never write an unknown feature in case round trips are a concern?

Reply:
@roeap looks like this worked, PTAL :)